functions/src/modules/SE.functions.ts

```diff
@@ -88,3 +93,276 @@ export const processNewAllotment = functions.database
+
+    let _import = false;
+ 
+
+    // [CHECK #1] File name belongs to the list identified by the sheet name       
+    if (helpers.extractListFromFilename(row['Audio File Name']) != sheetTitle) {
+        console.log("File name belongs to a different list.");
+        summary.misplaced.push({
+            audioFileName: row['Audio File Name'],
+            listName: sheetTitle,
+            errorMessage: `File name belongs to a different list.`
+        });
+    }
+
+    resolution = row['Resolution'];
+    fidCheckRes = row['Fidelity Check Resolution'];
+
+    if (resolution && fidCheckRes) {
+        resolution = resolution.toLowerCase();
+        fidCheckRes = fidCheckRes.toLowerCase();
+
+        // [CHECK #2] Both resolutions belong to the list of resolutions, case-insensitively.
+        if (resolutions.indexOf(resolution) < 0 || resolutions.indexOf(fidCheckRes) < 0) {
+            console.log(`Audio file name: ${row['Audio File Name']} -- Error: Resolution, or Fidelity check resolution values isn't valid`);
+            summary.invalidResoultion.push({
+                audioFileName: row['Audio File Name'],
+                resolution,
+                fidCheckRes,
+                errorMessage: `Resolution, or Fidelity check resolution values isn't valid.`
+            });
+        }
+
+        // [CHECK #3] Resolution and Fidelity Check Resolution should be equal, case-insensitively.
+        if (resolution !== fidCheckRes) {
+            console.log(`Audio file name: ${row['Audio File Name']} -- Error: Mismatch in Resolution and Fidelity check resolution values.`);
+            summary.mismatchedResolutions.push({
+                audioFileName: row['Audio File Name'],
+                resolution,
+                fidCheckRes,
+                errorMessage: `Mismatch in Resolution and Fidelity check resolution values.`
+            });
+        }
+
+        // [CHECK #6] Beginning and Ending should be filled for rows with Resolution equal to `OK`.
+        if (resolution === 'ok') {
+            _import = true;
+        }
+
+    } else // [Filter criterion #1] Rows not having (Resolution, or Fidelity check resolutin) are skipped SILENTLY
+        summary.noResolution.push({ audioFileName: row['Audio File Name'], resolution, fidCheckRes });
+
+    return _import;
+}
+
+
+
+export const importChunks = functions.runWith({
+    timeoutSeconds: 300,
+    memory: '2GB'
+}).pubsub.topic('daily-tick').onPublish(async (message) => {
+    const auth = await google.auth.getClient({
+        scopes: ["https://www.googleapis.com/auth/spreadsheets.readonly"]
+    });
+
+    const sheets = google.sheets({ version: "v4", auth });
+    const spreadsheetId = functions.config().reporting.content.processing.spreadsheet_id;
+    let sheetscMetadata  = await helpers.buildSheetIndex(sheets, spreadsheetId);
+
+
+    let currentFile, // used to increment the chunks of an audiofile coming from multiple rows into a single object
+        summary = {
```

Comment:
I think that it might be more logical to group the summary around the rows themselves. Something like
```
summary = {
  validityFailures: [{
    row: {}, //data
    issues: ["misplaced", "invalidResolution", …]
  }]
}